includes/admin/class-edd-notices.php

```diff
@@ -103,6 +103,74 @@ public function show_notices() {
 
 		}
 
+		/**
+		 * Notice for users running PHP < 5.6.
+		 * @since 2.10
+		 */
+		if ( version_compare( PHP_VERSION, '5.6', '<' ) && ! get_user_meta( get_current_user_id(), '_edd_upgrade_php_dismissed', true ) ) {
```

Comment:
@robincornett This should likely have the same capability check as the other server-related notices: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/77c88c0e43fb4c74ec2320dd73bd068d529080e1/includes/admin/class-edd-notices.php#L62

Showing this to Subscribers, Authors, etc wouldn't be relevant.